examples/examples/attrs-literal-field-names.rs

```diff
@@ -0,0 +1,22 @@
+#![deny(rust_2018_idioms)]
+
+use tracing::{debug, info, span, Level};
```

Comment:
Let's remove the `info` import, as we don't use it in the example:
```suggestion
use tracing::{debug, span, Level};
```

Reply:
done